ui/app/AppLayouts/Wallet/panels/SwapInputPanel.qml

```diff
@@ -277,6 +277,8 @@ Control {
                                                   { noSymbol: !amountToSendInput.inputIsFiat })
 
                 visible: d.isSelectedHoldingValidAsset && root.swapSide === SwapInputPanel.SwapSide.Pay
+                // FIXME: This should be enabled after #15709 is resolved
+                enabled: false
```

Comment:
The PR says that the button is hidden; otoh this just disables it. Second, I thought we agreed to keep it that way for Swap?

Reply:
In the discussion, we agreed to disable it instead of hiding it. 
because by only disabling it the user still can have access to the maximum value, which is pretty useful information.

@dlipicar , @saledjenic  would you please confirm if it should be disabled for both swapModel and sendModel?

Reply:
Also, you must first fix the QML UI tests if you want to disable it (at least for SwapModal). Another point, as we don't have a design for this usecase, are we fine with this looks of the disabled Max button? It looks weird imo:

![image](https://github.com/user-attachments/assets/79d82659-31cf-48e4-a780-d304d730188a)

Reply:
@belalshehab yeap, just disable, not hide. I think @caybro wants you to change the PR title

Reply:
> Also, you must first fix the QML UI tests if you want to disable it (at least for SwapModal). Another point, as we don't have a design for this usecase, are we fine with this appearance of the disabled Max button? It looks weird imo:
> 
> ![image](https://private-user-images.githubusercontent.com/5377645/351724729-79d82659-31cf-48e4-a780-d304d730188a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4MjY1ODYsIm5iZiI6MTcyMTgyNjI4NiwicGF0aCI6Ii81Mzc3NjQ1LzM1MTcyNDcyOS03OWQ4MjY1OS0zMWNmLTQ4ZTQtYTc4MC1kMzA0ZDczMDE4OGEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MjRUMTMwNDQ2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDZlNDU5NzAyNWFkZTU2MDhkMmFjMjMxZDc5NzU0NWM4YjZhMjVmYTU0NDI4MjlkOGE2MzYwMDhjNzFlNTk5OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.2Ffkr3Qp_6BVct7tyYS4k2W7peYyNxm-huZ4Zpxlhso)

yeah, the background color is barely visible... Can we get someone from Design to take a look?

Reply:
> > Also, you must first fix the QML UI tests if you want to disable it (at least for SwapModal). Another point, as we don't have a design for this usecase, are we fine with this appearance of the disabled Max button? It looks weird imo:
> > ![image](https://private-user-images.githubusercontent.com/5377645/351724729-79d82659-31cf-48e4-a780-d304d730188a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4MjY1ODYsIm5iZiI6MTcyMTgyNjI4NiwicGF0aCI6Ii81Mzc3NjQ1LzM1MTcyNDcyOS03OWQ4MjY1OS0zMWNmLTQ4ZTQtYTc4MC1kMzA0ZDczMDE4OGEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcyNCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MjRUMTMwNDQ2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDZlNDU5NzAyNWFkZTU2MDhkMmFjMjMxZDc5NzU0NWM4YjZhMjVmYTU0NDI4MjlkOGE2MzYwMDhjNzFlNTk5OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.2Ffkr3Qp_6BVct7tyYS4k2W7peYyNxm-huZ4Zpxlhso)
> 
> yeah, the background color is barely visible... Can we get someone from Design to take a look?

I could do the same thing with `opacity` that I did elsewhere for the middle "exchange"  button